setup.cfg

```diff
@@ -33,7 +32,7 @@ setup_requires =
 install_requires =
     tensorflow>=2.5,<2.8  # tensorflow.experimental.numpy
     tensorflow_probability>=0.11
-python_requires = >=3.6
+python_requires = >=3.7
```

Comment:
> It still can be run on Python 3.6, but it won't receive any updates.
> https://github.com/zfit/phasespace/pull/69#issuecomment-977976244

This should stay 3.6 then, right?

Reply:
I meant that just older version can still run on Python 3.6. and from 1.5 on not anymore (so anything < 1.5 will still run). So effectively dropping it.

Anyone using Python 3.6 will run into the problem that packages don't provide any updates anymore, phasespace (or tensorwaves) will be just one of them. So it's still usable, just with older versions

Reply:
Ok makes sense.